src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -440,6 +443,12 @@ type WebAppBuilder() =
     member this.EnableCi(state:WebAppConfig) = this.SourceControlCi(state, Enabled)
     [<CustomOperation "disable_source_control_ci">]
     member this.DisableCi(state:WebAppConfig) = this.SourceControlCi(state, Disabled)
+    [<CustomOperation "tags">]
```

Comment:
Rename to `add_tags` and `add_tag` please, unless each operation will overwrite the existing tags that have already been set.